saleor/graphql/account/types.py

```diff
@@ -335,15 +339,18 @@ def resolve_note(root: models.User, info):
     )
     @traced_resolver
     def resolve_events(root: models.User, info):
-        return root.events.all()
+        return CustomerEventsByUserLoader(info.context).load(root.id)
 
     @staticmethod
     @traced_resolver
     def resolve_orders(root: models.User, info, **_kwargs):
-        viewer = info.context.user
-        if viewer.has_perm(OrderPermissions.MANAGE_ORDERS):
-            return root.orders.all()
-        return root.orders.non_draft()  # type: ignore
+        def _resolve_orders(orders):
+            requester = info.context.user
+            if requester.has_perm(OrderPermissions.MANAGE_ORDERS):
+                return orders
```

Comment:
Shouldn't we also return all orders for `app` with `MANAGE_ORDERS`?

Reply:
Good catch Iga! 💪 
I believe, the behaviour for apps and staff users should be the same.